client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -223,7 +223,7 @@ export const HoldList = ( { context, holds, isMarketplace, isPlaceholder, transl
 
 	return (
 		<>
-			{ ! isPlaceholder && context !== 'plugin-details' && (
+			{ ! isPlaceholder && context !== 'plugin-details' && context !== 'themes' && (
```

Comment:
Non-blocking: Looks like ``context !== 'plugin-details' && context !== 'themes' ` ` condition is used in a few places. Would it make sense to extract `context !== 'plugin-details' && context !== 'themes' ` into a function so that we don't duplicate this condition in a lot of places? This way, it would give some semantic meaning?

e.g. something like `isContextEligibleForWarnings(context)` would be easier to understand (at least for me) than using the condition everywhere; plus we can add some docblocks about it's purpose.

Reply:
The semantic meaning isn't "is context eligible for warnings" - warnings do get displayed still, they just get displayed differently (see [the issue](https://github.com/Automattic/dotcom-forge/issues/1299#issuecomment-1396767198)). HardBlockingNotice.

I'm not keen on most of the 'context' uses inside the component, it's a bit of a kludge. Perhaps ideally it would be broken down differently and we'd have a ThemeEligiblityWarnings and a PluginEligibilityWarnings etc components.

Reply:
Or maybe we'd just push the theme / plugin specific texts down into a child component and have EligibilityWarnings just pass the specific texts through 🤔 

Either way I think this is deserving of some refactoring and thinking in future.